benchmarks/simple_mustang.ex

```diff
@@ -6,8 +6,8 @@ defmodule SimpleMustang do
     page = browser |> Playwright.Browser.new_page()
     _response = Playwright.Page.goto(page, options.target_url)
 
-    :ok = Playwright.Page.fill(page, "[name=display_name]", "stampede")
-    :ok = Playwright.Page.click(page, "[type=submit]")
+    :ok = Playwright.Page.fill(page,  "(//input[@id=\"display_name\"])[2]", "stampede")
```

Comment:
IMO This seems like a nasty hack. The problem is due to 2 types of inputs - one for mobile view and one for desktop. Usually, the are in sync and allow smooth transitions between views. I'd solve it a bit differently: by adding an id to the wrapping `div`s

in `assets/src/features/home-page/components/VideoroomHomePage.tsx`
on line 176: add `id="mobile_home_form"`
on line 193: add `id="desktop_home_form"`

and then in this selector:
```suggestion
    :ok = Playwright.Page.fill(page,  "#desktop_home_form [name=display_name]", "stampede")
```

Side note: IMO the actual inputs should have different id's, but it's not that straightforward to fix